python/lsst/meas/astrom/match_probabilistic_task.py

```diff
@@ -0,0 +1,174 @@
+# This file is part of meas_astrom.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import lsst.afw.geom as afwGeom
+import lsst.geom as geom
+import lsst.pipe.base as pipeBase
+import lsst.utils as utils
+from .matcher_probabilistic import MatchProbabilisticConfig, MatcherProbabilistic
+
+import logging
+import numpy as np
+import pandas as pd
+from typing import Dict, Set, Tuple
+
+__all__ = ['MatchProbabilisticTask']
+
+
+class MatchProbabilisticTask(pipeBase.Task):
+    """Run MatchProbabilistic on a reference and target catalog covering the same tract.
+    """
+    ConfigClass = MatchProbabilisticConfig
```

Comment:
Are there no configs specific to the task? Would it make more sense to have the prob matcher be a configurable subtask of the main task? It's fine if not.

Reply:
Just to clarify, I could have folded the Matcher into the Task instead of having a separate class, but this way the Matcher has no stack dependencies besides pex_config (which I believe can be installed as a standalone package itself), and in principle can be imported independently without building the package. Whether that ever ends up being useful to anyone remains to be seen.

Reply:
Sounds good. Like I said on the other repo, I think you already take care of this by having the configurable in the pipeline class.